atomix/cluster/src/main/java/io/atomix/raft/partition/RaftPartition.java

```diff
@@ -247,7 +247,7 @@ private boolean shouldStepDown() {
   }
 
   public CompletableFuture<Void> goInactive() {
-    return server.goInactive();
+    return server.stop();
   }
```

Comment:
:question: `goInactive` is used in two places, on unrecoverable errors and on recoverable errors on followers. Is it intended that recoverable errors on followers now stop the raft server?
Either way, I think we should rename the method to `stop` or similar.

Reply:
Previously it was going inactive in both places, which was intended to behave similar to stop. So we are not changing the expected behavior.